### PR TITLE
Harmonize docs generation

### PR DESCRIPTION
**Pull request by ericbottard**
Fixes spring-cloud/spring-cloud-dataflow-server-kubernetes#90

### CODE REVIEW

**Comment by trisberg** [comment]
I'm getting merge conflicts and not sure what changes should go in and which shouldn't

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -104,7 +106,7 @@ Configure the Data Flow server URI with the following command (use the IP addres
  | |_| | (_| | || (_| | |  _| | | (_) \ V  V /    / / / / / /
  |____/ \__,_|\__\__,_| |_|   |_|\___/ \_/\_/    /_/_/_/_/_/
 
-1.0.0.BUILD-SNAPSHOT
+{dataflow-project-version}
```
Comment (ilayaperumalg) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc:109]: This should be the `{project-version}` as it points to the `dataflow-server-kubernetes`.

**Comment by ilayaperumalg** [comment]
LGTM, merging along with the changes required for https://github.com/spring-cloud/spring-cloud-dataflow-server-kubernetes/pull/91#discussion_r68715589

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc (hunk 1 above)
Comment (ilayaperumalg) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc:109]: @ericbottard pointed out that this is supposed to be the dataflow server core version only. Hence, this is correct as is.

**Comment by ilayaperumalg** [comment]
Cherry-picked the changes and merged as 73c8aea918ce99aca6eaa2593eda9f65bd9bedfe. Also verified the generated doc with these changes.

**Pull request closed without merging**